DrinkManagerWeb/Views/Drinks/DrinkDetails.cshtml

```diff
@@ -28,6 +28,9 @@
                 }
             </div>
         </div>
+        <div class="p-2">
+            <partial name="Partials/BackButton"/>
+        </div>
```

Comment:
On responsive view this button isn't at the bottom of the page. It's in the middle. Also I think this button should have the same width as Ingredients box.

Reply:
Done!